lib/vector-vrl/web-playground/public/index.js

```diff
@@ -92,14 +97,12 @@ export class VrlWebPlayground {
 
     addVersions() {
         let vectorLinkElement = document.getElementById('vector-version-link');
-        let vectorVersion = vector_version();
```

Comment:
This call was redundant since we cached the result above. Now the UI load faster.